docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -0,0 +1,209 @@
+
+## Protocol and convenience method examples
+
+The coding patterns and types used by client library protocol and convenience methods vary slightly based on whether the library depends on `Azure.Core` or `System.ClientModel`. The differences primarily influence the .NET types used for handling request and response data.
+
+### Libraries that depend on Azure.Core
+
+Many Azure SDK client libraries depend on the `Azure.Core` library. For example, the [`Azure.AI.ContentSafety`](/dotnet/api/overview/azure/ai.contentsafety-readme) library depends on the `Azure.Core` library and provides a `ContentSafetyClient` class that exposes both protocol and convenience methods.
+
+#### Work with protocol methods
+
+The following code uses a `ContentSafetyClient` to call the `AnalyzeText` protocol method:
+
+```csharp
+// Create the client
+var safetyClient = new ContentSafetyClient(
+    new Uri("<content-safety-service-uri>"),
+    new DefaultAzureCredential());
+
+// Create the message content
+RequestContent message = RequestContent.Create(new
+    {
+        text = "What is Microsoft Azure?",
+    });
+
+// Call the protocol method
+Response response = safetyClient.AnalyzeText(
+    message, 
+    new RequestContext()
+    {
+         ErrorOptions = ErrorOptions.NoThrow;
+    });
+
+// Display the response data
+using (StreamReader streamReader = new StreamReader(response.ContentStream))
+{
+    Console.WriteLine(streamReader.ReadToEnd());
+}
+```
+
+The preceding code demonstrates the following protocol method patterns:
+
+- Uses the `RequestContent` type as a  to supply data for the request body.
+- Uses the `RequestContext` type to configure request options.
+- Returns data using the `Response` type.
+- Reads the `ContentStream` to access the response data.
+
+#### Work with convenience methods
+
+The following code uses a `ContentSafetyClient` to call the `AnalyzeText` convenience method:
+
+```csharp
+// Create the client
+var safetyClient = new ContentSafetyClient(
+    new Uri("content-safety-service-uri"),
+    new DefaultAzureCredential());
+
+// Call the convenience method
+AnalyzeTextResult result = safetyClient.AnalyzeText("What is Microsoft Azure?");
+
+// Display the response data
+foreach (var item in result.CategoriesAnalysis)
+{
+    Console.Write($"{item.Category}: ");
+    Console.Write(item.Severity);
+    Console.WriteLine();
+}
+```
+
+The preceding code demonstrates the following `Azure.Core` convenience method patterns:
+
+- Uses a standard C# primitive or model type as a parameter.
+- Returns a friendly C# type that represents the result of the operation.
+
+### Libraries that depend on System.ClientModel
+
+Some Azure client libraries depend directly on the `System.ClientModel` library. For example, the [`OpenAI`](https://www.nuget.org/packages/OpenAI/2.0.0-beta.7) library depends on the `System.ClientModel` library and provides a `ChatClient` class that exposes both protocol and convenience methods.
```

Comment:
OpenAI actually isn't an Azure client library -- Azure.AI.OpenAI is because it talks to the Azure OpenAI service. But the OpenAI client library talks to the third-party OpenAI service.

Reply:
Given this, another suggestion for framing this -- given that SCM is intended primarily for unbranded clients -- maybe something like ... _client libraries that talk to non-Azure services are being created that use patterns similar to the ones developed for Azure.Core-based libraries.  For example, the OpenAI client that talks to the OpenAI services.  These libraries are based on a library called SCM that has patterns similar to Azure.Core._

I don't feel like I nailed that, but I am happy to let you run with it from there.